site/content/docs/5.2/examples/form/index.html

```diff
@@ -0,0 +1,233 @@
+                class="overflow-visible">
+                <use xlink:href="/docs/{{< param docs_version >}}/assets/img/boosted-sprite.svg#user"></use>
+              </svg>
+              <span class="visually-hidden">My account</span>
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </nav>
+</header>
+<main class="bd-content order-1">
+  <div class="bg-white title-bar">
+    <div class="container-xxl d-block">
+      <div class="row">
+        <div class="col-sm-12 col-lg-5 offset-lg-3">
+          <h1 class="display-1">Become a reseller</h1>
+        </div>
+      </div>
+    </div>
+  </div>
+  <div class="container-xxl pt-4">
+    <div class="row">
+      <div class="col-sm-12 col-lg-5 offset-lg-3">
+        <p class="ll-sm h3 mb-4">You are a professional and wish to resell Orange products to your
+          customers? Please fill the form and we will get back to you shortly.
+        </p>
+        <form class="needs-validation mb-5" novalidate>
+          <div class="row">
+            <div class="col-12">
+              <p class="fw-bold mb-4">
+                <span class="text-primary">*</span> indicates required
+              </p>
+            </div>
+            <div class="col-sm-4 col-md-6 mb-3">
+              <label for="selectTitle" id="selectTitleLabel" class="form-label is-required">Title</label>
+              <select class="form-select" id="selectTitle" aria-labelledby="selectTitleLabel selectTitleFeedback"
+                 autocomplete="sex" required>
+                <option selected disabled value="" aria-hidden="true">Select a title</option>
+                <option value="1">Miss</option>
+                <option value="2">Ms.</option>
+                <option value="3">Mrs.</option>
+                <option value="3">Mr.</option>
+              </select>
+              <div class="valid-feedback">
+                Looks good!
+              </div>
+              <div id="selectTitleFeedback" class="invalid-feedback">
+                Please select a valid title.
+              </div>
+            </div>
+            <div class="col-12">
+              <div class="mb-3">
+                <label for="inputFirstName" id="inputFirstNameLabel" class="form-label is-required">First name</label>
+                <input type="text" class="form-control" id="inputFirstName"
+                  aria-labelledby="inputFirstNameLabel inputFirstNameFeedback" autocomplete="given-name" required>
+                <div class="valid-feedback">
+                  Looks good!
+                </div>
+                <div id="inputFirstNameFeedback" class="invalid-feedback">
+                  Please enter a valid first name.
+                </div>
+              </div>
+              <div class="mb-3">
+                <label for="inputLastName" id="inputLastNameLabel" class="form-label is-required">Last name</label>
+                <input type="text" class="form-control" id="inputLastName"
```

Comment:
There's an "issue" with the ids in this form regarding autocomplete. We could use the examples in https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete to name the ids so that autocomplete really suggest things. We have more chance to have the same name as other websites if we use for example here `lastName` instead of `inputLastName`. Same thing for other inputs.

Reply:
Not sure about this one. `autocomplete` is not linked to the inputs `id` but to the personal inforprmations stored in the browser.

Reply:
Firefox / macOS with `inputLastName` nothing is suggested. With `lastName` (don't remember if the `name=` was required) I have some suggestions.